superset/connectors/sqla/views.py

```diff
@@ -470,7 +474,7 @@ def post_add(  # pylint: disable=arguments-differ
             )
 
     def post_update(self, item: "TableModelView") -> None:
-        self.post_add(item, flash_message=False)
+        self.post_add(item, flash_message=False, fetch_metadata=False)
```

Comment:
I'm not sure why we were syncing metadata on `post_update`, so I disabled it.

Reply:
@dpgaspar do you know why we call `post_add` every time we `post_update` the dataset?  Previously we were calling `fetch_metadata` and `create_table_permissions` every time we updated, but after this change we'll only call `create_table_permissions`. Is this just to make sure any potentially stale perms get synced?

Reply:
To update the permissions if the dataset table_name or database change, yet if that is done it will leave a stale permission name behind

Reply:
Ok, then I'll just leave this like this, as I believe we won't be hitting `post_edit` with the new CRUD.